src/justfile.rs

```diff
@@ -179,6 +189,13 @@ impl<'a> Display for Justfile<'a> {
         write!(f, "\n\n")?;
       }
     }
+    for (alias_name, recipe_name) in &self.aliases {
```

Comment:
I don't have an opinion really. I rarely do a `--dump`, I usually just use `--list` to see what's available and go from there. I changed `Display` to show aliases after assignments and before `recipes`. 

While I was changing this, I thought it would be nice if `--list` can show aliases alongside recipes, what do you think?